packages/frontend/src/components/LastDeploymentCard.jsx

```diff
@@ -30,7 +30,7 @@ export default observer(() => {
               ui.isMobile
             )}
             <a
-              href={`https://goerli.arbiscan.io/address/0x${BigInt(
+              href={`${info.network.explorer}/address/0x${BigInt(
```

Comment:
this should be padding to length 40 
```ts
0x${BigInt(attesterId).toString(16).padStart(40, '0')}
```

Reply:
why we need to padStart?

Reply:
0x0023.. should be different from 0x23..
If it is not padding 0s

Reply:
Only the href link need to padStart to lead to correct explorer page, right?
For the display, if there's so many 0, then after `shortenId`, they will look so alike (e.g.`0x00...123`)

Reply:
yes